### ISSUE TITLE
`customRangeButtonLabel` is not used

### ISSUE THREAD

**Issue by Dwightojo** [valor-software/ng2-bootstrap#6384]
**Description:**
The field `customRangeButtonLabel` in the `bs-datepicker.config.ts` is currently not used for anything. the custom range button does not exist. Until this functionality is added i recommend removing the field from the config as it is quite confusing at the moment.

**Comment by valorkin** [comment]
yeah, I event had a design for it, but never had a chance to actually add it

**Comment by valorkin** [comment]
oh, actually it's used 
![image](https://user-images.githubusercontent.com/1107171/147362544-67e8da75-3d2d-404e-81c3-4505ef797355.png)

**Comment by valorkin** [comment]
```
  /**
   * Predefined ranges
   */
  ranges?: BsCustomDates[];
```

**Comment by valorkin** [comment]
here is the demo https://valor-software.com/ngx-bootstrap/#/components/datepicker?tab=overview#quick-select-ranges

**Comment by valorkin** [comment]
nah actually you right, need to recall what I was planning for it :/

**Comment by Dwightojo** [comment]
Hey, no problem. and yeah i got turned around by it too.
